velox/exec/LocalPlanner.cpp

```diff
@@ -407,6 +407,8 @@ std::shared_ptr<Driver> DriverFactory::createDriver(
   ctx->driver = driver.get();
   std::vector<std::unique_ptr<Operator>> operators;
   operators.reserve(planNodes.size());
+  driver->minOutputBatchRows_ =
```

Comment:
This is an anti-pattern. Please, remove minOutputBatchRows_ variable from Driver class and use driverCtx->queryConfig().xxx instead.

Reply:
done